File: android/app/src/main/java/com/example/fugliara/ccs002/server/dataobjects/OrderItem.java
package com.example.fugliara.ccs002.server.dataobjects;

import java.util.List;

public class OrderItem {
    @com.google.gson.annotations.SerializedName("id")
    public String id;
    @com.google.gson.annotations.SerializedName("routeItem")
    public RouteItem route;
    @com.google.gson.annotations.SerializedName("userItem")
    public UserItem user;
    @com.google.gson.annotations.SerializedName("dateTime")
    public String date;
    @com.google.gson.annotations.SerializedName("comments")
    public String comments;
    @com.google.gson.annotations.SerializedName("productDetailItems")
    public List<ProductDetailItem> productDetailItems;

    public OrderItem() {
        id = "";
        route = null;
        user = null;
        date = "";
        comments = "";
        productDetailItems = null;
    }
}

File: azure/transporterSolution/transporterService/DataObjects/OrderItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using transporterService.DataTranspObjects;

namespace transporterService.DataObjects
{
    public partial class OrderItem : BaseItem
    {
        public OrderItem()
        {
            ProductDetailItems = new List<ProductDetailItem>();
        }
        public virtual UserItem UserItem { get; set; }
        public virtual RouteItem RouteItem { get; set; }
        public string DateTime { get; set; }
        public string Comments { get; set; }

        public virtual ICollection<ProductDetailItem> ProductDetailItems { get; set; }
    }
}